WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -140,6 +146,33 @@ extension SurveyViewController: WKNavigationDelegate {
         removeLoadingView()
     }
 }
+// MARK: Survey Tags
+//
+internal extension URL {
+    func tagPlatform() -> URL {
```

Comment:
For sure! I've added some tests and refactored a little bit the function so it can be better described and easier to test, the changes are submitted here: https://github.com/woocommerce/woocommerce-ios/pull/3041/commits/2982507f542c5eaff3e81d4062f880e001618e4e and here: https://github.com/woocommerce/woocommerce-ios/pull/3041/commits/4f4e3e670898705367a71f0181d8e8b87a82f24e. Let me know if that's a desirable approach or not! Thank you for the suggestion!